### DISCUSSION TITLE
Connection error issue

### DISCUSSION THREAD

**Discussion by cyrusyn** [kubilus1/autoortho#386] [General]
<img width="735" alt="image" src="https://github.com/kubilus1/autoortho/assets/56527139/81aeb48f-ef79-4af1-851d-0e8db36124b4">

I have a solid connection via ethernet cabble. Firewall has been chcked both up and down side and both ports are open for this app. Need a help resolving this issue. Keeps happening after reinstalling the app and all continents again and again.

Both dokan and winfsp are installed.

**Comment by cyrusyn** [comment]
Tried reinstalling it and made sure firewalls had exceptions on autoortho but still this.
<img width="655" alt="image" src="https://github.com/kubilus1/autoortho/assets/56527139/8701a82c-cdc0-4b6a-a469-1f5b95a502b7">
<img width="662" alt="image" src="https://github.com/kubilus1/autoortho/assets/56527139/2eca4764-13cf-4600-89d1-0fe3552ff528">
<img width="467" alt="image" src="https://github.com/kubilus1/autoortho/assets/56527139/12dea1ce-dfbb-4301-8d39-e4b5e2fb45b9">

**Reply by Jeff2122** [reply]
Try a different data source? It is trying but not getting data.
USGS is sh1t for me. Try BI or Firefly.
You can preview the data for most of the sources online. Just google them.

**Reply by cyrusyn** [reply]
Changed to Bing and out of 8 attempts, I got success in 2 attempts. But even these two attempts took some time due to missing chunks or failed chunks.

**Comment by zippy77777** [comment]
I am having the same issue, nothing is working
